pkg/controller/infrastructure/terraform_reconciler_test.go

```diff
@@ -165,6 +165,7 @@ var _ = Describe("Actuator", func() {
 			tf.EXPECT().Apply(ctx)
 			tf.EXPECT().GetStateOutputVariables(ctx, gomock.Any())
 			tf.EXPECT().GetRawState(ctx).Return(tfState, nil)
+			tf.EXPECT().GetRawState(ctx).Return(tfState, nil)
```

Comment:
TODO?

Reply:
No, this was due to `GetRawState` being called twice. Admittedly, that was not the ideal solution.

I've moved the call to `GetRawState` up and passed the result to the functions reducing it back to one expected call.